### PR TITLE
Adds accepted talks, adds times according to current schedule.

### PR DESCRIPTION
**Pull request by jvzammit**
## This PR

* Adds accepted talks.
* Adds times according to current schedule.
* Adds media images + twitter handle for each speaker.
* Missing "programme events" such as "venue open" and "lunch break" -- added work in progress in #17

### CODE REVIEW

File: themes/djangoday/layouts/shortcodes/talklist.html [hunk 1]
```diff
@@ -20,7 +20,7 @@ <h2>
   <img src="{{ $new.Permalink }}" alt="Speaker" class="img-speaker float-right" />
 
   <h2 class="no-link-styling">
-  <!--<span class="talktime">{{ dateFormat "15:04" .Params.talk_date }}</span>-->
+  <span class="talktime">{{ dateFormat "15:04" .Params.talk_date }}</span>
```
Comment (benjaoming) [inline themes/djangoday/layouts/shortcodes/talklist.html]: Can we comment this one out again? That way, we can maintain the time schedule without announcing it :)

**Review by benjaoming — APPROVED** [review]
It looks good to go! Am just waiting for a reconfirmation from the speakers, and we can add them :)

File: themes/djangoday/layouts/shortcodes/talklist.html (hunk 1 above)
Comment (jvzammit) [reply themes/djangoday/layouts/shortcodes/talklist.html]: Makes sense @benjaoming thanks, pushed. You can merge if it looks good to you.

File: content/talks/michael/_index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+img_name: "michael.jpg"
+remote: false
+layout: "single"
+stage: true
+twitter: "michjnich"
+description: "How 12 factor app helped Divio migrate over 10k Postgres instances"
+# youtube_id: "zAKcwo5Cyw8"
+# slides: "https://docs.google.com/presentation/d/1OTI--ZQLLR3N8ixl4OktEwbXfiau_0BNXicl_3j5uYc/edit?usp=sharing"
+# notes: "andrew.jpeg"
+published: true
```
Comment (benjaoming) [inline content/talks/michael/_index.md]:
```suggestion
published: false
```

**Review by benjaoming — APPROVED** [review]

**Pull request merged**